### PR TITLE
Remove pmt to simplify

### PR DESCRIPTION
**Pull request by brianmcmichael**
Removes the optional lump payment to simplify code. This functionality can be handled by minting tokens in the executive.

### CODE REVIEW

File: src/DssVest.sol [hunk 1]
```diff
@@ -87,35 +87,30 @@ contract DssVest {
...
         require(_bgn < block.timestamp + MAX_VEST_PERIOD, "dss-vest/bgn-too-far");
         require(_tau > 0,                                 "dss-vest/tau-zero");
         require(_tau <= MAX_VEST_PERIOD,                  "dss-vest/tau-too-long");
         require(_clf <= _tau,                             "dss-vest/clf-too-long");
-        require(_pmt < uint128(-1),                       "dss-vest/payout-error");
-        require(_pmt <= _amt,                             "dss-vest/bulk-payment-higher-than-amt");
 
         id = ++ids;
-        if (_amt - _pmt != 0) {      // safe because pmt <= amt
+        if (_amt != 0) {      // safe because pmt <= amt
```
Comment (naszam) [inline src/DssVest.sol:103]: I think the comment `// safe because pmt <= amt` can be removed

Comment (brianmcmichael) [reply src/DssVest.sol]: Fixed!

**Review by hexonaut — APPROVED** [review]
LGTM. Yes agree with removing this.

**Review by naszam — APPROVED** [review]
LGTM

**Pull request merged**